tmt/steps/prepare/install.py

```diff
@@ -461,12 +461,9 @@ def install_from_repository(self) -> None:
                         Command(package)
                         )
                 except tmt.utils.RunError as error:
-                    if error.stderr \
-                            and "error: Packages not found" in error.stderr \
-                            and self.skip_missing:
-                        self.warn(f"No match for recommended package '{package}'.")
-                        continue
-                    raise
+                    self.debug(f"Package installation failed: {error}")
+                    self.warn(f"No match for recommended package '{package}'.")
```

Comment:
Could it be something more generic? It can be missing dependency / conflict ...

Reply:
Ok, message updated in d971b094.